### PR TITLE
Add document to describe the process of deploying Wordpress on Carrier

### LINKED ISSUE #144: Deploy wordpress (no db)
We need to check if the PHP buildpack we are using is capable of running Wordpress. This story doesn't include the setup of Wordpress which comes after deployment.

### PR DESCRIPTION
**Pull request by jimmykarily**
[Fixes #144]

### CODE REVIEW

File: docs/wordpress.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+The PHP buildpack supports additional ini files for PHP through
+[the PHP_INI_SCAN_DIR mechanism](https://paketo.io/docs/buildpacks/language-family-buildpacks/php/#php_ini_scan_dir).
+
+We need zlib and mysqli extensions enabled:
+
+```bash
+mkdir .php.ini.d
+cat << EOF > .php.ini.d/extensions.ini
+extension=zlib
+extension=mysqli
```
Comment (andreas-kupries) [inline docs/wordpress.md:44]: `EOF` missing.

File: docs/wordpress.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+```
+
+## Deploy
+
+```
+carrier push wordpress
```
Comment (andreas-kupries) [inline docs/wordpress.md:50]: Should we remove the tarball from the cwd first ? I don't think that it is required, right ?

File: docs/wordpress.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+```
+
+## Additional steps
+
+Wordpress needs a database to work. After visiting the route of your deployed
+application you will have to set the connection details to the database.
+
+You can install a MySQL database on your cluster or use an external one. One
+option is using a helm chart like this one: https://bitnami.com/stack/mysql/helm
```
Comment (andreas-kupries) [inline docs/wordpress.md:59]: IMHO adding the steps for setting up the database would be helpful too, while we do not have general services support.

**Review by andreas-kupries — CHANGES_REQUESTED** [review]
Nits, and (maybe) a bit more information at the end (about the database).

File: docs/wordpress.md (hunk 1 above)
Comment (jimmykarily) [reply docs/wordpress.md:44]: Added.

File: docs/wordpress.md (hunk 2 above)
Comment (jimmykarily) [reply docs/wordpress.md:50]: Removed.

File: docs/wordpress.md (hunk 3 above)
Comment (andreas-kupries) [reply docs/wordpress.md:61]: Retracting, the linked page has the necessary docs.

**Review by andreas-kupries — APPROVED** [review]

**Pull request merged**